models/i18n.cfc

```diff
@@ -309,7 +307,7 @@ component singleton accessors="true" {
 	 * @thisTimeFormat FULL=0, LONG=1, MEDIUM=2, SHORT=3
 	 * @tz timezone
 	 */
-	string function i18nDateTimeFormat(
+	string function DateTimeFormat(
```

Comment:
can you please make sure all method names start with lower case camel case. I think running cfformat will update everything for ya

Reply:
nope, cfformat doesn't update this. So I did it manually